cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -182,6 +182,39 @@ def _validate_curie_ontology(self, term_id: str, column_name: str, allowed_ontol
                 f"'{term_id}' in '{column_name}' is not a valid ontology term id of '{', '.join(allowed_ontologies)}'."
             )
 
+    def _validate_curie_str(self, term_str: str, column_name, curie_constraints: dict):
+        """
+        Validate a curie str based on some constraints. If there are any errors, it adds them to self.errors
+
+        :param str term_str: the curie term str to validate
+        :param str column_name: Name of the column in the dataframe
+        :param dict curie_constraints: constraints for the curie term to be validated,
+        this part of the schema definition
+
+        :rtype None
+        """
+        # If there are exceptions and this is one then skip to end
+        if "exceptions" in curie_constraints and term_str in curie_constraints["exceptions"]:
```

Comment:
moved from '_validate_curie' because exceptions are only valid if they are the only term in the string, not if they are part of a comma-delimited str